wideq/client.py

```diff
@@ -432,7 +432,15 @@ def _get_config(self, key):
             self.device.id,
             key,
         )
-        return json.loads(base64.b64decode(data).decode('utf8'))
+        data = base64.b64decode(data).decode('utf8')
+        try:
+            return json.loads(data)
+        except json.decoder.JSONDecodeError:
+            LOGGER.warning(
+                'JSONDecodeError malformed json (%s)', data)
```

Comment:
Not sure we need to include the full data here, and I don't think it needs to be a warning because the second-try decoding might work. Can we just log (in debug mode) "attempting to fix JSON format"?